src/ScottPlot/plottables/IPlottable.cs

```diff
@@ -1,19 +1,20 @@
-﻿using ScottPlot.Drawing;
+﻿using ScottPlot.Config;
+using ScottPlot.Drawing;
 using System;
-using System.Collections.Generic;
 using System.Drawing;
-using System.Text;
 
 namespace ScottPlot
 {
-    // TODO: Work toward strangling the Plottable class by impementing this interface 
-    //       as plottables are refactored. Once no more plottables inherit from Plottable,
-    //       move its abstract methods into this interface.
+    [Obsolete("Use IPlottable", true)]
+    public abstract class Plottable { }
+
     public interface IPlottable
     {
         void Render(PlotDimensions dims, Bitmap bmp, bool lowQuality = false);
-
-        bool IsValidData(bool deepValidation = false);
-        string ValidationErrorMessage { get; }
+        string ErrorMessage(bool deepValidation = false);
```

Comment:
It is very unclear that this is a method with a bunch of checks. At first glance, it looks a lot like a simple query for a string. Maybe you should come up with a more descriptive name?